src/main/java/Duke/TaskList/tasks/Deadlines.java

```diff
@@ -0,0 +1,20 @@
+package Duke.TaskList.tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+
+public class Deadlines extends Task {
```

Comment:
Should the class name be singular instead of plural? I think there are some other class names that could be singular, too.